libs/dh/admin/feature-user-management/src/lib/shared/dh-user-roles.component/dh-user-roles.component.html

```diff
@@ -30,22 +30,43 @@
           *ngTemplateOutlet="
             userRolesHeadline;
             context: {
-              $implicit: numberOfRoles$ | push
+              $implicit:
+                (selectMode ? numberOfAssignableRoles$ : numberOfSelectedRoles$)
+                | push
             }
           "
         ></ng-container>
       </watt-card-title>
-      <div class="userRoles" *ngFor="let org of userRoleView.organizations">
+      <div
+        class="userRoles"
+        [ngClass]="{ selectMode: selectMode }"
+        *ngFor="let org of userRoleView.organizations"
+      >
         <ng-container *ngFor="let actor of org.actors">
           <h6>
             {{ org?.name }} | GLN {{ actor.actorNumber }} |
```

Comment:
suggestion: consider moving `GLN {{ actor.actorNumber }}` to a translation that takes `actorNumber` as a param.